package.json

```diff
@@ -31,6 +31,7 @@
     "react-transition-group": "2.5.0",
     "redux": "4.0.0",
     "redux-thunk": "2.3.0",
+    "reselect": "4.0.0",
```

Comment:
Reselect will be used in the next steps. (optimizations - tree is a bit laggy at the moment)